src/extension.ts

```diff
@@ -311,32 +311,35 @@ export function activate(context: VSCode.ExtensionContext) {
   context.subscriptions.push(openRuleCommand);
 
   const updateServerStorageCommandCallback = () => {
-    updateServerStorage();
-    updateProjectBinding();
+    updateServerStorage().then(() => {
+      updateProjectBinding().then(() => {
```

Comment:
I think you can write (it might be a bit clearer) : 
```ts
updateServerStorage()
   .then(updateProjectBinding)
   .then(() => {
      VSCode.window.showInformationMessage("Successfully updated SonarLint server storage");
   });
```

And now that I think about it, if you don't care about the call order of those two functions and they can be run in parallel, you could use something like: 
```ts
Promise.all([updateServerStorage(), updateProjectBinding()]).then(() => {
   VSCode.window.showInformationMessage("Successfully updated SonarLint server storage");
});
```